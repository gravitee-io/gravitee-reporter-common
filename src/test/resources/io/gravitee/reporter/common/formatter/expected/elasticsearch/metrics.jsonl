{ "index" : { "_index" : "gravitee-http.Metrics-2023.08.28", "_id" : "13f5ae30-068b-4e2d-b5ae-30068bae2d2d", "pipeline" : "my-pipeline"} }
{ "gateway":"gateway-id" ,"@timestamp":"2023-08-18T11:16:21.941Z" ,"transaction":"13f5ae30-068b-4e2d-b5ae-30068bae2d2d" ,"method":3 ,"uri":"/test" ,"status":200 ,"response-time":150 ,"api-response-time":144 ,"proxy-latency":6 ,"request-content-length":0 ,"response-content-length":275 ,"plan":"e115ea63-7cef-4646-95ea-637cef7646ec" ,"api":"ff3c6c48-53e0-41d6-bc6c-4853e011d656", "api-name": "API Name" ,"application":"91f077b0-1204-49e4-b077-b0120419e4f6" ,"local-address":"127.0.0.1" ,"remote-address":"127.0.0.1" ,"endpoint":"https://api.gravitee.io/echo" ,"path":"" ,"host":"localhost:8082" ,"user-agent":"curl/7.88.1" ,"security-type":"API_KEY" ,"security-token":"21b560b2-59b8-4a4b-921a-32b3731fdec4" ,"subscription":"04975880-f147-43bc-9758-80f147e3bcbb" ,"custom": { "zone":"europe-north1-a" } }
