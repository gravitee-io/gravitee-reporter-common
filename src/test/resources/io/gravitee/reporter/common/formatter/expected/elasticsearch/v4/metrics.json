{
  "type": "v4-metrics",
  "date": "2023.08.18",
  "_id": "076aea69-6024-4590-aaea-6960247590a0",
  "gateway": "gateway-id",
  "@timestamp": "2023-08-18T11:46:53.844Z",
  "request-id": "076aea69-6024-4590-aaea-6960247590a0",
  "client-identifier": "318e47e5-349c-4fa4-8e47-e5349c3fa444",
  "transaction-id": "076aea69-6024-4590-aaea-6960247590a0",
  "api-id": "5f67b38f-0700-4557-a7b3-8f0700855779",
  "plan-id": "8463511c-fbed-4ca9-a351-1cfbed9ca99d",
  "application-id": "91f077b0-1204-49e4-b077-b0120419e4f6",
  "subscription-id": "318e47e5-349c-4fa4-8e47-e5349c3fa444",
  "http-method": 3,
  "local-address": "127.0.0.1",
  "remote-address": "127.0.0.1",
  "host": "localhost:8082",
  "uri": "/test-v4",
  "path-info": "",
  "user-agent": "",
  "request-ended": "true",
  "endpoint": "https://api.gravitee.io/echo",
  "endpoint-response-time-ms": 137,
  "status": 200,
  "response-content-length": 274,
  "gateway-response-time-ms": 144,
  "gateway-latency-ms": 7
}
